app/csv.go

```diff
@@ -0,0 +1,25 @@
+package app
+
+import (
+	"encoding/csv"
+	"fmt"
+	"os"
+)
+
+func CSVReader() [][]string {
+
+	// open the file
+	csvFile, err := os.Open("../pokedex.csv")
```

Comment:
It would be even better to not hardcode this path here, we could define were the file path is from a variable, or as a param, or from a config file

Reply:
I mention this, because when doing unit testing, we're coupled to this specific file if we leave it like that, but we would like to read different files to account for multiple scenarios